modules/nw-dns-operator.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+// * networking/dns/dns-operator.adoc
+
+[id="dns-operator-{context}"]
+= DNS Operator
+
+The DNS Operator implements the `dns` API from the `operator.openshift.io` API
+group. The Operator manages a DNS Service for the cluster, and configures
```

Comment:
I don't think "Operator" or "Service" in this sentence need to be capitalized.

Reply:
I changed Operator to operator, but I kept Service capitalized since I 'm referring to a Kubernetes Service. Let me know if you have recommendations on how to reference Kubernetes API types.

Reply:
If it is referring to the API type, the convention used elsewhere in openshift-docs is to use backticks: ``` `Service` ```.